openvpn/tunnelUp.sh

```diff
@@ -40,6 +40,6 @@ if [ "${PEER_DNS}" != "no" ]; then
 fi
 
 /etc/transmission/start.sh "$@"
-[[ ! -f /opt/tinyproxy/start.sh ]] || /opt/tinyproxy/start.sh
+[[ ! -f /opt/privoxy/start.sh ]] || /opt/privoxy/start.sh
```

Comment:
I know you just updated paths here, but I think this line will be much clearer with the reversed condition.
Also the test doesn't check if the file is executable or not:
```suggestion
[[ -f /opt/privoxy/start.sh && -x /opt/privoxy/start.sh ]] && /opt/privoxy/start.sh
```